lib/medic/store.rb

```diff
@@ -113,5 +115,32 @@ def disable_all_background_delivery(block=Proc.new)
       })
     end
 
+  private
+
+    def prepare_for_save(sample)
```

Comment:
Also, it looks like there are [predefined metadata keys](https://developer.apple.com/library/ios/documentation/HealthKit/Reference/HealthKit_Constants/index.html#//apple_ref/doc/constant_group/Metadata_Keys) that we will need to parse at some point.